packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions/components/CheckBoxInputs/components/CheckBoxInput.js

```diff
@@ -15,11 +15,16 @@ export const StyledBox = styled(Box)`
     opacity: 0.01;
     position: absolute;
   }
+
+  &:focus-within {
+    box-shadow: 0 0 2px 2px ${props => props.theme.global.colors['accent-1']};
```

Comment:
Non-blocking: the theme has a designation for a focus color at `theme.global.colors.focus` which right now is mapping to `accent-1`, but if we change it, then this wouldn't need to be updated in that case.

Reply:
forgot about that, always good to set programmatically 👍